addOns/automation/src/main/javahelp/org/zaproxy/addon/automation/resources/help/contents/job-spider.html

```diff
@@ -43,7 +43,7 @@ <H2>YAML</H2>
         statistic: 'automation.spider.urls.added'       # String: Name of an integer / long statistic, currently supported: 'automation.spider.urls.added'
         operator: '>='                                  # String ['==', '!=', '>=', '>', '<', '<=']: Operator used for testing
         value: 100                                      # Int: Change this to the number of URLs you expect to find
-        onFail: 'info'                                  # String [warn, error, info]: Change this to 'warn' or 'error' for the test to take effect
+        onFail:                                         # String: One of 'warn', 'error', 'ignore', mandatory
```

Comment:
Why change the default value? Its mandatory, so will fail if people dont add it, thats why the default value was there :)

Reply:
Unresolving the conversation as I cant see why the default was removed :)